book/.gitignore

```diff
@@ -0,0 +1,2 @@
+stabilizer-manual
+src/firmware
```

Comment:
```suggestion
/stabilizer-manual
/src/firmware
```

I think.

Reply:
Also needed a trailing slash - thanks for catching this one.